README.md

```diff
@@ -42,4 +42,13 @@ $ pq --msgtype com.example.dog.Dog --stream varint <./tests/samples/dog_stream
 }
 ```
 
-[More usage](./USAGE.md)
+### Compiling for Windows
```

Comment:
I would probably move this to a `WINDOWS.md` file with a `[Windows build instructions](./WINDOWS.md)` link - but we can save that for later.